api/serverless.ts

```diff
@@ -237,13 +237,22 @@ const serverlessConfiguration: AWS = {
   functions: {},
 };
 
-serverlessConfiguration.custom = {
-  ...serverlessConfiguration.custom,
-  config: {
-    application: "${ssm:/config/application}",
-    infrastructure: "${ssm:/config/infrastructure}",
-  },
-};
+serverlessConfiguration.custom =
+  stage === "local"
+    ? {
+        ...serverlessConfiguration.custom,
+        config: {
+          application: "${ssm:/config/application}",
+          infrastructure: "${ssm:/config/infrastructure}",
+        },
+      }
+    : {
+        ...serverlessConfiguration.custom,
+        config: {
+          application: "${ssm:/${self:provider.stage}/config/application}",
+          infrastructure: "${ssm:/config/infrastructure}",
+        },
+      };
```

Comment:
We were adding this as a short-term fix to get the testing site up. Do we actually want this included going forward?

Reply:
We can discuss this sometime tomorrow. But i think there are ways we can go about this. We would have to update the current config/application across the 3 environments to reference the stages so that the change here would be minimal. (Meaning we update the /config/application in dev to be /dev/config/application). This would also apply to staging and prod